recipe/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "sis-extras" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sis_extras-{{ version }}.tar.gz
+  sha256: e7e1892ab5eef36e38880dc9b636597dce83162df4d51f3f7f509a7e792e8a73
+  patches:
+    - patches/0001-loosen-Python-and-streamlit-constraints.patch
+    - patches/0002-add-a-timeout-to-the-test-run.patch
+
+build:
+  # Strictly, upstream is py==38 and we don't have
+  # snowflake-snowpark-python for 3.12
+  # no streamlit on s390x
+  skip: True  # [py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pdm-backend
+    - pip
+  run:
+    - python
+    # Strictly, upstream is streamlit==1.31.1
+    - streamlit
+    - snowflake-snowpark-python >=1.13.0
+    - plotly >=5.20.0
+
+test:
+  source_files:
+    - tests
+  imports:
+    - sis_extras
+  requires:
+    - pip
+    - pytest
+    # sqlparse can be used if available
+    - sqlparse
+  commands:
+    - pip check
+    - pytest -v tests/test_sis_extras.py
+    - python -m tests.test_example_app
+
+about:
+  home: https://github.com/blackary/sis-extras
+  summary: Python package to make it easy to develop beautiful, performant streamlit-in-snowflake apps
+  description: |
+    The sis-extras package provides a set of Python utilities designed
+    to help you develop beautiful, performant streamlit-in-snowflake
+    apps.
+  license: MIT
+  license_url: https://spdx.org/licenses/MIT.html
```

Comment:
```suggestion
  license_file: LICENSE.txt
```

Reply:
@ifitchet

Reply:
lol -- missed this comment, pushed by a separate commit

Reply:
Duh!   builds failing because this change hasn't been released!

Reply:
Right, I haven't thought about that.

They merged the PR on the repository, but the pypi archive won't contain this. You could add the raw file from the github repository, adding a source+sha